test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1830,3 +1904,17 @@ func addKubeSystemPdbs(f *framework.Framework) (func(), error) {
 	}
 	return cleanup, nil
 }
+ func createPriorityClasses(f *framework.Framework) func() {
```

Comment:
This may be personal preference, but I feel like using a map here would be cleaner:

```
  priorityClasses = map[string]int32{
    expendablePriorityClassName: -15,
    lowPriorityClassName: 100,
    highPriorityClassName: 1000,
  }
```

```
for className, priority := range priorityClasses {
	_, err := f.ClientSet.SchedulingV1alpha1().PriorityClasses().Create(&v1alpha1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: className}, Value: priority})
 	Expect(err == nil || errors.IsAlreadyExists(err)).To(Equal(true))
}

return func() {
  for className, _ := priorityClasses{
     f.ClientSet.SchedulingV1alpha1().PriorityClasses().Delete(className, nil)
  }
}
```

Reply:
Changed